cddl/int-version-map.cddl

```diff
@@ -0,0 +1,5 @@
+int-version-map = {
+  &(version: 0): version-int-text,
+  &(version-scheme: 1): &(multipartnumeric: 1)
```

Comment:
```suggestion
  &(version-scheme: 1): &(decimal: 4)
```

Reply:
I believe decimal fits better for SEV-SNP version, as it doesn't have multiple parts. [RFC 9393](https://www.rfc-editor.org/rfc/rfc9393.html#name-version-scheme) also mentions about multipartnumeric overlapping with decimal, which would be the case for SEV-SNP.

Reply:
The "floating point number" bothered me, but if it works fine for integers, sure.